sdk/identity/Azure.Identity/src/ManagedIdentityClient.cs

```diff
@@ -57,7 +57,7 @@ public ManagedIdentityClient(ManagedIdentityClientOptions options)
 
         public async ValueTask<AccessToken> AuthenticateAsync(bool async, TokenRequestContext context, CancellationToken cancellationToken)
         {
-            AuthenticationResult result = await _msal.AcquireTokenForClientAsync(context.Scopes, context.TenantId, async, cancellationToken).ConfigureAwait(false);
+            AuthenticationResult result = await _msal.AcquireTokenForClientAsync(context.Scopes, context.TenantId, context.IsCaeEnabled, async, cancellationToken).ConfigureAwait(false);
```

Comment:
My only concern is that you may want it to be consistent with other requests that do support CAE. I am leaning towards keeping it consistent with the other credentials just to avoid confusion.